src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,67 @@
+package duke;
+
+import duke.command.Command;
+
+import java.io.IOException;
+
+/**
+ * Represents the Duke chatbot that stores users tasks.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for the Duke chatbot.
+     * @param filePath filepath to save the tasks on.
+     */
```

Comment:
```suggestion
    /**
     * Constructor for the Duke chatbot.
     *
     * @param filePath filepath to save the tasks on.
     */
```
I think that it may be better to leave an empty line between the description and parameter section so that it looks more organized and neat (Its part of the coding standard).